e2etest/pkg/frr/container/container.go

```diff
@@ -111,9 +158,24 @@ func UpdateContainerVolumePermissions(containerName string) error {
 	}
 
 	cmd := fmt.Sprintf("chown -R %d:%d %s", uid, gid, frrMountPath)
-	out, err := exec.Command("docker", "exec", containerName, "sh", "-c", cmd).CombinedOutput()
+	out, err := exec.Command("docker", "exec", c.Name, "sh", "-c", cmd).CombinedOutput()
+	if err != nil {
+		return errors.Wrapf(err, "Failed to change %s container volume permissions. %s", c.Name, string(out))
+	}
+
+	return nil
+}
+
+// Updating the BGP config file.
+func (c *Frr) UpdateBGPConfigFile(bgpConfig string) error {
```

Comment:
why did you move the function? might look confusing in the log

Reply:
done